org.eclipse.winery.frontends/app/tosca-management/src/app/model/enums.ts

```diff
@@ -29,6 +29,7 @@ export enum ToscaTypes {
     RelationshipType = 'relationshiptypes',
     ArtifactType = 'artifacttypes',
     ArtifactTemplate = 'artifacttemplates',
+    DeploymentArtifacts = 'deploymentArtifacts',
```

Comment:
This is wrong. I guess we have to use the TableType enum everywhere instead of the ToscaType.